src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int GROW_FACTOR = 2;
+    private int size;
+    private Node<K,V>[] table = new Node[DEFAULT_CAPACITY];
 
     @Override
     public void put(K key, V value) {
-
+        resizeCheck();
+        int index = getIndex(key);
+        Node<K, V> node = new Node<>(key, value);
+        if (table[index] == null) {
+            table[index] = node;
+            size++;
+            return;
+        }
+        Node<K, V> kvNode = findNode(index, key);
+        if (kvNode != null) {
+            kvNode.value = value;
+            return;
+        }
+        for (Node<K,V> i = table[index]; i != null; i = i.next) {
+            if (i.next == null) {
+                i.next = node;
+                size++;
+                return;
+            }
+        }
```

Comment:
you have a second loop in this method (first inside findNode). try to refactor your code so only one loop is used